src/components/Card/PlaylistsCard/PlaylistsCard.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styles from "./playlists-card.module.css";
+import { RiPlayList2Fill } from "react-icons/ri";
+import { FaFileVideo } from "react-icons/fa";
+import { Link } from "react-router-dom";
+import { emptyBox, playlist } from "assets";
+
+export function PlaylistsCard({
+  playlist: { _id: playlistId, title, description, videos },
+}) {
+  console.log(videos);
```

Comment:
remove console.log

Reply:
Thanks for pointing it out. Will remove it.